code/modules/admin/verbs/randomverbs.dm

```diff
@@ -663,8 +663,17 @@ Traitors and the like can also be revived with the previous role mostly intact.
 		return
 
 	if(job_master)
+		var/currentpositiontally
+		var/totalpositiontally
+		to_chat(src, "<span class='notice'>Job Name: Filled job slot / Total job slots <b>(Free job slots)</b> </span>")
 		for(var/datum/job/job in job_master.occupations)
-			to_chat(src, "[job.title]: [job.total_positions]")
+			to_chat(src, "<span class='notice'>[job.title]: [job.current_positions] / \
+			[job.total_positions == -1 ? "<b>UNLIMITED</b>" : job.total_positions] \
+			 <b>([job.total_positions == -1 ? "UNLIMITED" : job.total_positions - job.current_positions])</b></span>")
+			if(!(job.total_positions == -1))
+				currentpositiontally += job.current_positions
+				totalpositiontally += job.total_positions
+		to_chat(src, "<b>Currently filled job slots (Excluding unlimited): [currentpositiontally] / [totalpositiontally] ([totalpositiontally - currentpositiontally])")

```

Comment:
naked `to_Chat` & you never close your bold tag.

Reply:
Closing the tag - Though I am keeping it black instead of blue following other admin message's convention. (And for visual distinction)

Reply:
Yes thats fine, keep it black. You still need to span it. Remember we talked about the reason why? If you're looking to ever easily dark theme, you need everything remaining easily changable.... go to browserOutput.css and find a good class for it....

Reply:
This isn't resolved :(

We're just going to have to go and manually fix this later if we go down this route.